src/models/PullRequest.js

```diff
@@ -0,0 +1,32 @@
+class PullRequest {
+  constructor(pullRequest) {
+    this.pullRequest = pullRequest;
+  }
+
+  id() {
+    return this.pullRequest.id;
+  }
+
+  number() {
```

Comment:
@michaelmurray6298 btw this is a shared method that exists on the prototype, as I was suggesting above for the `fetchRepos` method inside `Org`.